client/src/shared/utils/navBarList.ts

```diff
@@ -1,7 +1,44 @@
 export const navItems = [
-  { url: '/dashboard', text: 'Dashboard' },
-  { url: '/courses', text: 'Courses' },
-  { url: '/learning_paths', text: 'Learning Paths' }
+  { url: '/', text: 'Dashboard', dropdownItems: [] },
+  {
+    url: '/',
+    text: 'Training',
+    dropdownItems: [
+      { text: 'Courses', url: '/trainer/courses/list' },
+      { text: 'Learning Paths', url: '/' }
+    ]
+  },
+  {
+    url: '/',
+    text: 'Training Elements',
+    dropdownItems: [
+      { text: 'Materials', url: '/element/material' },
+      { text: 'Studio', url: '/' },
+      { text: 'Assessments', url: '/' },
+      { text: 'Checklists', url: '/' },
+      { text: 'Knowledge Checks', url: '/' },
+      { text: 'Instructor-Led Training', url: '/' }
+    ]
+  },
+  {
+    url: '/',
+    text: 'Users and Groups',
+    dropdownItems: [
+      { text: 'Users', url: '/' },
+      { text: 'Groups', url: '/' },
+      { text: 'Permission and Roles', url: '/' }
+    ]
+  },
+  {
+    url: '/',
+    text: 'Reporting',
+    dropdownItems: [
+      { text: 'Dashboard', url: '/' },
+      { text: 'Reporting Wizard', url: '/' },
+      { text: 'Report Builder', url: '/' },
+      { text: 'Advanced Reports', url: '/' }
+    ]
+  }

```

Comment:
Please revert back all these changes, because we are not using this structure anymore, there will be only 3 routes, and please only update url that is related to your tasks
In your case you only need to change the url for courses, and I think it should be `/trainer/courses` in line 3 
![image](https://github.com/framgia/sph-lms/assets/110364690/8c90e5bf-0c1c-45cf-8f69-76917eb78534)

Reply:
Like I commented above, please add your route ( `/trainer/courses`) to the line 2 (please alter red rectangle with your route)